packages/lesswrong/lib/collections/posts/constants.ts

```diff
@@ -31,3 +36,16 @@ export const postStatusLabels = [
     label: 'deleted'
   }
 ];
+
+const amaTagIdSetting = new DatabasePublicSetting<string | null>('amaTagId', null)
+const openThreadTagIdSetting = new DatabasePublicSetting<string | null>('openThreadTagId', null)
+const startHerePostIdSetting = new DatabasePublicSetting<string | null>('startHerePostId', null)
+
+export const tagSettingIcons = new Map<DatabasePublicSetting<string | null>, React.ComponentType<React.SVGProps<SVGElement>>>([
```

Comment:
Minor point of habit: This API (`Map`) is in the reference class of APIs-to-check-the-compatibility-matrix-on-Mozilla Developer Network-when-using. (Since it runs on the client, doesn't come from an `import`, and isn't something with a ton of preexisting usages). In this case it works in everything except IE11, and we're not supporting IE11 anyways.

Reply:
Good to know, thanks!